src/collision.cpp

```diff
@@ -623,8 +623,10 @@ bool collision_check_intersection(Environment *env, IGameDef *gamedef,
 		Collision detection
 	*/
 	aabb3f checkbox = box_0;
-	checkbox.MinEdge += pos_f;
-	checkbox.MaxEdge += pos_f;
+	// aabbox3d::intersectsWithBox(box) returns true when the faces are touching perfectly.
+	// However, we do not want want a true-ish return value in that case. Add some tolerance.
```

Comment:
Alternatively I could implement `intersectsWithBoxButFacesExclusive(box)` where checks are performed with `>` instead of `>=`. Conveniently, the tolerance also compensates for any floating point imprecisions that *might* still occur (needs testing).

Reply:
> this change sounds potentially far-reaching

`collision_check_intersection` is only used by the sneak glitch "fix glitch through ceiling" thing though.

This behavior needs documentation in the header file.

Reply:
@grorp Documented in 04fd8aea8.